src/pystatis/config.py

```diff
@@ -33,6 +33,46 @@
     "zensus": re.compile(r"^\d{4}[A-Z]-\d{4}$"),
     "regio": re.compile(r"^((\d{5}-.{1,2}($|-.*$))|(A.*$)|([0-9A-Z]{10}$)|(\d{5}\w-Z-\d{1,2}))"),
 }
+COLUMN_NAME_DICT = {
```

Comment:
maybe change into something more speaking like `LANG_TO_COL_MAPPING` but definitely better ;) The name should indicate that the variable holds column names by language or a mapping between the two